spanner/api/Spanner.Samples/UpdateDatabaseAsync.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+// [START spanner_update_database]
```

Comment:
So I checked this with the spanner team and the response I got was :

>The region tag "spanner_update_database" is correct. It was finalized to show the API usage instead of feature usage in the sample.
This comment could help you
https://docs.google.com/document/d/1ljn-WGUgPfowmFO5RWsyPtndiMSdAsteYCWzvkr9QA4/edit?disco=AAAApltwJWQ

Reply:
Thanks.